### DISCUSSION TITLE
Does ccmenu2 only support add cctray project one by one?

### DISCUSSION THREAD

**Discussion by flniu** [erikdoe/ccmenu2#21] [Q&A] [answered]
ccmenu1 supports select and add multiple cctray projects in batch.

But add cctray project in ccmenu2 can only choose one project:
![image](https://github.com/user-attachments/assets/7e0a5911-7aa6-4c03-b53b-7161918a65e7)

**Comment by erikdoe** [comment, marked as answer]
Yes, apologies, this is an oversight and a regression. It's tracked in #14.
